set up ant design styling for register page, added basic styling to form

DanielleKoduru opened the pull request:
# Short Description

Added AntD styling to Register page 

## Rationale/purpose of the feature

For the user to have ease as they navigate around the application 

## Link to relevant User Story

[AntD register styling](https://trello.com/c/hCU9oOwZ/13-register-page)

## Checklist for author and reviewers

- [ ] Commits are well named and descriptive
- [ ] PR is focused on a single problem, is small enough and easy to review
- [ ] PR requests were sent to multiple reviewers
- [ ] PR is reviewed by more than one team member
- [ ] Code is consistently formatted
- [ ] Code is suitably commented on
- [ ] There is no duplicate code
- [ ] Code tries to follow best practices
- [ ] Decide with reviewers if this PR should have tests before merging

DanielleKoduru committed:
set up ant design styling for register page, added basic styling to form

DanielleKoduru committed:
added styling to register page

DanielleKoduru committed:
styling for register complete

clarejackson reviewed (approved):
This is code looks great! I can't wait to see what you have been able to do.

DanielleKoduru committed:
clean up

DanielleKoduru committed:
Working on clean up
Merge branch' of https://github.com/Girl-Squad-Essentialism/FrontEnd into feature/Register-Styling

barbaralois reviewed (approved):
Form set-up looks great along with general file clean-up/organization
